chapter_13_database_layer_validation.asciidoc

```diff
@@ -760,7 +778,10 @@ class ListViewTest(TestCase):
 Note that the `NewItemTest` class disappears completely.
 I've also changed the name of the redirect test to make it explicit that it only applies to POST
 requests.
+// DAVID did you? Looks like the same name to me?
+
 
+// DAVID: we switched to running the unit test, would be good to provide the command.
```

Comment:
are you still going through and doing all the listings on your own pc?  it should be obvious, we've just edited a unit test file.  i expect readers not to need too much hand-holding at this point in the book...